puppet/zulip/manifests/static_asset_compiler.pp

```diff
@@ -17,10 +17,13 @@
     }
     'redhat': {
       $static_asset_compiler_packages = [
-        # TODO CentOS doesn't have closure-compiler
         'yuicompressor',
         'gettext',
       ]
+      $setup_closure_compiler_file = "${::zulip_scripts_path}/lib/setup-closure-compiler"
+      exec{'setup_closure_compiler':
+        command => "bash -c '${setup_closure_compiler_file}'",
+      }
```

Comment:
We removed the dependency of `closure-compiler` in master in the last week or so, so I think you can just drop this commit.